### DISCUSSION TITLE
弹窗选择取消，窗口无法关闭

### DISCUSSION THREAD

**Discussion by HelloJasonZ** [zuihou/zuihou-admin-cloud#326] [01.常见问题]
![image](https://github.com/dromara/lamp-cloud/assets/20830000/5df45525-8b96-4985-90c0-962bb6667702)
当弹窗窗口，选择取消的话，窗口不会关闭
![image](https://github.com/dromara/lamp-cloud/assets/20830000/23715fab-b846-4e2e-8214-f56a32dc85f3)

**Comment by zuihou** [comment]
什么浏览器， F12 控制台打开看看点取消报错了没

**Reply by HelloJasonZ** [reply]
google浏览器，F12点击取消没有出现报错。我是将vite升级到[升级vite版本到4.2.1]，并按照gitlab更新了首屏加载。点击确认能跳转。
部分的package.json文件内容


"dependencies": {
    "@ant-design/colors": "^7.0.0",
    "@ant-design/icons-vue": "^6.1.0",
    "@iconify/iconify": "^3.1.0",
    "@logicflow/core": "^1.2.1",
    "@logicflow/extension": "^1.2.1",
    "@vue/runtime-core": "^3.2.47",
    "@vueuse/core": "^9.13.0",
    "@zxcvbn-ts/core": "^2.2.1",
    "ant-design-vue": "^3.2.16",
    "axios": "^1.3.4",
    "codemirror": "^6.0.1",
    "cropperjs": "^1.5.13",
    "crypto-js": "^4.1.1",
    "dayjs": "^1.11.7",
    "echarts": "^5.4.1",
    "lodash-es": "^4.17.21",
    "mockjs": "^1.1.0",
    "nprogress": "^0.2.0",
    "path-to-regexp": "^6.2.1",
    "pinia": "^2.0.33",
    "print-js": "^1.6.0",
    "qrcode": "^1.5.1",
    "qs": "^6.11.1",
    "resize-observer-polyfill": "^1.5.1",
    "showdown": "^2.1.0",
    "sortablejs": "^1.15.0",
    "tinymce": "^5.10.7",
    "vditor": "^3.9.1",
    "vue": "^3.2.47",
    "vue-i18n": "^9.2.2",
    "vue-json-pretty": "^2.2.4",
    "vue-router": "^4.1.6",
    "vue-types": "^5.0.2",
    "xlsx": "^0.18.5",
    "vxe-table": "^4.3.5",
    "vxe-table-plugin-antd": "^3.0.6",
    "vxe-table-plugin-export-xlsx": "^3.0.4",
    "sass": "^1.57.1",
    "xe-utils": "^3.5.6",
    "js-base64": "^3.7.2",
    "driver.js": "^0.9.8",
    "@vue/shared": "^3.2.33",
    "vue3-blocks-tree": "^0.6.1",
    "highlight.js": "^11.5.1",
    "vuedraggable": "^4.1.0",
    "vue-codemirror": "^6.0.2",
    "@vueuse/shared": "^8.4.2",
    "@codemirror/lang-html": "^6.1.0",
    "@codemirror/lang-java": "^6.0.0",
    "@codemirror/lang-javascript": "^6.0.2",
    "@codemirror/lang-json": "^6.0.0",
    "@codemirror/lang-markdown": "^6.0.1",
    "@codemirror/lang-sql": "^6.0.0",
    "@codemirror/lang-xml": "^6.0.0",
    "@codemirror/theme-one-dark": "^6.0.0",
    "exceljs": "^4.3.0"
  },
  "devDependencies": {
    "@commitlint/cli": "^17.5.0",
    "@commitlint/config-conventional": "^17.4.4",
    "@iconify/json": "^2.2.39",
    "@purge-icons/generated": "^0.9.0",
    "@types/crypto-js": "^4.1.1",
    "@types/fs-extra": "^11.0.1",
    "@types/inquirer": "^9.0.3",
    "@types/lodash-es": "^4.17.7",
    "@types/mockjs": "^1.0.7",
    "@types/node": "^18.15.7",
    "@types/nprogress": "^0.2.0",
    "@types/qrcode": "^1.5.0",
    "@types/qs": "^6.9.7",
    "@types/showdown": "^2.0.0",
    "@types/sortablejs": "^1.15.1",
    "@typescript-eslint/eslint-plugin": "^5.56.0",
    "@typescript-eslint/parser": "^5.56.0",

    "@vitejs/plugin-vue": "^4.1.0",
    "@vitejs/plugin-vue-jsx": "^3.0.1",
    "@vue/compiler-sfc": "^3.2.47",
    "@vue/test-utils": "^2.3.1",
    "autoprefixer": "^10.4.14",
    "conventional-changelog-cli": "^2.2.2",
    "cross-env": "^7.0.3",
    "cz-git": "^1.6.1",
    "czg": "^1.6.1",
    "dotenv": "^16.0.3",
    "eslint": "^8.36.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-vue": "^9.10.0",
    "esno": "^0.16.3",
    "fs-extra": "^11.1.1",
    "husky": "^8.0.3",
    "increase-memory-limit": "^1.0.7",
    "inquirer": "^9.1.5",
    "less": "^4.1.3",
    "lint-staged": "^13.2.0",

    "picocolors": "^1.0.0",
    "postcss": "^8.4.21",
    "postcss-html": "^1.5.0",
    "postcss-less": "^6.0.0",
    "prettier": "^2.8.7",
    "rimraf": "^4.4.1",
    "rollup": "^3.20.2",
    "rollup-plugin-visualizer": "^5.9.0",
    "stylelint": "^15.3.0",
    "stylelint-config-recommended": "^11.0.0",
    "stylelint-config-recommended-vue": "^1.4.0",
    "stylelint-config-standard": "^31.0.0",
    "stylelint-order": "^6.0.3",
    "terser": "^5.16.6",
    "ts-node": "^10.9.1",
    "typescript": "^5.0.2",
    "unplugin-vue-setup-extend-plus": "^0.4.9",
    "vite": "^4.3.5",
    "vite-plugin-compression": "^0.5.1",
    "vite-plugin-mkcert": "^1.13.4",
    "vite-plugin-mock": "^2.9.6",
    "vite-plugin-progress": "^0.0.6",
    "vite-plugin-purge-icons": "^0.9.2",
    "vite-plugin-pwa": "^0.14.6",
    "vite-plugin-style-import": "^2.0.0",
    "vite-plugin-svg-icons": "^2.0.1",
    "vite-vue-plugin-theme": "^1.0.0",
    "vite-vue-plugin-html": "^1.0.1",
    "vue-eslint-parser": "^9.1.0",
    "vue-tsc": "^1.2.0",
    "vite-plugin-windicss": "^1.8.10"
  },
  "resolutions": {
    "bin-wrapper": "npm:bin-wrapper-china",
    "rollup": "^3.20.0",
    "gifsicle": "5.2.0"
  },
  "engines": {
    "node": "> 16.18.0"
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": [
      "eslint --fix",
      "prettier --write"
    ],
    "{!(package)*.json,*.code-snippets,.!(browserslist)*rc}": [
      "prettier --write--parser json"
    ],
    "package.json": [
      "prettier --write"
    ],
    "*.vue": [
      "eslint --fix",
      "prettier --write",
      "stylelint --fix"
    ],
    "*.{scss,less,styl,html}": [
      "stylelint --fix",
      "prettier --write"
    ],
    "*.md": [
      "prettier --write"
    ]
  },
  "config": {
    "commitizen": {
      "path": "node_modules/cz-git"
    }
  }

**Reply by zuihou** [reply]
你可以试试： https://datasource.tangyh.top/ ，看能不能取消， 演示环境 vite:"^4.3.5"。 

按你说的，你可以尝试一下还原vite版本，确认是否vite升级导致的

**Reply by HelloJasonZ** [reply]
我回滚到了升级前的版本，和切换其他没升级的分支，依然发现弹出未关闭。是哪块有缓存吗？

**Comment by zuihou** [comment]
没缓存，代码在这，基本就对ant-design-vue Modal.confirm的简单封装，有问题可能就是ant-design-vue Modal的问题

<img width="1038" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/1ec5da38-0651-4f84-8d77-90d51373300f">

**Reply by HelloJasonZ** [reply]
那可能就是ant-design-vue Modal问题了，我试了案列里面的“消息提示”，点击取消和确认都无法消息。有其他的解决办法吗？
![image](https://github.com/dromara/lamp-cloud/assets/20830000/c776bed7-47c5-45a3-b170-53f3d12ab5fe)

**Comment by HelloJasonZ** [comment]
终于解决问题了，是vue 版本与antd-vue版本问题了。固定 Vue 3.3.4 ; antd-vue 固定 3.2.15 就能正常销毁了
https://github.com/vueComponent/ant-design-vue/issues/7239
